prow/lib.sh

```diff
@@ -160,13 +160,13 @@ function setup_kind_cluster() {
   fi
 
   # Create KinD cluster
-  if ! (kind create cluster --name="${NAME}" --config "${CONFIG}" --loglevel debug --retain --image "${IMAGE}" --wait=60s); then
+  if ! (kind create cluster --name="${NAME}" --config "${CONFIG}" --retain --image "${IMAGE}" --wait=60s); then
     echo "Could not setup KinD environment. Something wrong with KinD setup. Exporting logs."
     exit 1
   fi
 
-  KUBECONFIG="$(kind get kubeconfig-path --name="${NAME}")"
-  export KUBECONFIG
+  # KUBECONFIG="$(kind get kubeconfig-path --name="${NAME}")"
```

Comment:
why?

Reply:
This call is not required anymore in current kind versions. kind automatically sets $KUBECONFIG after successfully creating a cluster. I changed this to actually remove the code instead of commenting